lib/include/ert/ecl/ecl_kw_magic.h

```diff
@@ -291,16 +291,19 @@ values (2e20) are denoted with '*'.
 */
 
 
-#define INTEHEAD_UNIT_INDEX     2
-#define INTEHEAD_NX_INDEX       8
-#define INTEHEAD_NY_INDEX       9
-#define INTEHEAD_NZ_INDEX      10
-#define INTEHEAD_NACTIVE_INDEX 11
-#define INTEHEAD_PHASE_INDEX   14
-#define INTEHEAD_DAY_INDEX     64
-#define INTEHEAD_MONTH_INDEX   65
-#define INTEHEAD_YEAR_INDEX    66
-#define INTEHEAD_IPROG_INDEX   94
+#define INTEHEAD_UNIT_INDEX      2
+#define INTEHEAD_NX_INDEX        8
+#define INTEHEAD_NY_INDEX        9
+#define INTEHEAD_NZ_INDEX       10
+#define INTEHEAD_NACTIVE_INDEX  11
+#define INTEHEAD_PHASE_INDEX    14
+#define INTEHEAD_DAY_INDEX      64
+#define INTEHEAD_MONTH_INDEX    65
+#define INTEHEAD_YEAR_INDEX     66
+#define INTEHEAD_TIMESTEP_INDEX 67
+#define INTEHEAD_REPORT_STEP    68
+#define INTEHEAD_IPROG_INDEX    94
+#define INTEHEAD_REPORT_INDEX   219 (report_steps-1)
```

Comment:
Ehhh - that is obviosuly a mistake; I guess it has "worked" because the `INTEHEAD_REPORT_INDEX` symbol has never been used.

Reply:
Fixed here: https://github.com/Statoil/libecl/pull/256

Reply:
> Fixed here: #256

:+1: 

By the way, since you're the expert, do you know of a setting&mdash;perhaps a flag in `INTEHEAD` or `LOGIHEAD`&mdash;that identifies whether or not the oil PVT table in the INIT file uses the `PVCDO` format?

Reply:
I'm sorry - but no; I am not aware of such a flag.

Reply:
> I'm sorry - but no; I am not aware of [a flag to indicate `PVCDO`]

Okay, thanks.  I have a few cases locally that **suggest** `LOGIHEAD(39)` (one-based indexing) could be related to this, but I don't have cases that change only the oil property table between `PVDO` and `PVCDO` to substantiate that claim.